fastlane/Fastfile

```diff
@@ -458,15 +469,61 @@ platform :ios do
-    ios_update_metadata(options)
-    # Jetpack: use the custom setup. Notice that we don't need extra steps to
-    # download `.strings` because Jetpack uses the same codebase as WordPress.
+    # Download `Localizable.strings` translations used within the app
+    parent_dir_for_lprojs = File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources')
+    ios_download_strings_files_from_glotpress(
+      project_url: 'https://translate.wordpress.org/projects/apps/ios/dev/', # TODO: Use a constant to DRY stuff
+      # TODO: Fill this in
+      locales: { gp1: lproj1, gp2: lproj2, … }, # TODO: Provide a big constant for this list somewhere. (See `ALL_LANGS` from `update-translation.rb`)
+      download_dir: parent_dir_for_lprojs
+    )
+
+    # Then redispatch the appropriate subset of translations back to the `InfoPlist.strings` and `Sites.strings` files in corresponding `*.lproj` dirs
+    parent_dir_for_intents_target_lprojs = File.join('WordPress', 'WordPressIntents')
+    ios_extract_keys_from_strings_files(
+      source_parent_dir: parent_dir_for_lprojs
+      target_original_files: [
+        File.join(parent_dir_for_lprojs, 'en.lproj', 'InfoPlist.strings'),
+        File.join(parent_dir_for_intents_target_lprojs, 'en.lproj', 'Sites.strings'),
+      ]
+    )
+
+    # Finally, also download the AppStore metadata (app title, keywords, etc.)
+    # @FIXME: Replace this whole lane with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
+    download_wordpress_localized_app_store_metadata
     download_jetpack_localized_app_store_metadata
   end
 
+  #####################################################################################
+  # download_wordpress_localized_app_store_metadata
+  # -----------------------------------------------------------------------------------
+  # Downloads localized metadata (intended for App Store Connect) from GlotPress for WordPress.
+  # -----------------------------------------------------------------------------------
+  # Usage:
+  # bundle exec fastlane download_wordpress_localized_app_store_metadata
+  #####################################################################################
+  desc 'Downloads localized WordPress metadata (for App Store Connect) from GlotPress'
+  lane :download_wordpress_localized_app_store_metadata do
+    # @FIXME: Replace this whole lane with a call to the future replacement of `gp_downloadmetadata` once it's implemented in the release-toolkit (see paaHJt-31O-p2).
+
+    # No need to `cd` into `fastlane` because of how Fastlane manages its paths internally.
+    sh './download_metadata.swift wordpress'
+
+    # @TODO: Make the `fastlane/metadata/en-US/release_notes.txt` path be the source of truth for the original copies in the future.
+    # (will require changes in the `update_appstore_strings` lane, the Release Scenario, the MC tool to generate the announcement post…)
```

Comment:
Can't wait to do this to simplify the whole thing greatly, as discussed in P2.

But that might require some parallel PRs to make sure such a change is reflected in all the places that depends on that path, including outside the repo and fastlane (Scenario, MC…), so we'll probably keep that for a later step.

Reply:
> keep that for a later step

Seems wise. I'm not too "worried" about the scenarios as much as I am for MC. But even there, if the file is not found we'll just end up with empty release notes which we can fill manually. That's to say: No need to sweat the parallel PRs coordination too much.